aws-api-payload.json

```diff
@@ -3,7 +3,7 @@
         "StepConcurrencyLevel": 1,
         "TerminationProtected": false,
         "BootstrapActions": [],
-        "LogUri": "s3://$ADG_S3_LOG_BUCKET/emr/aws-ingestion/",
+        "LogUri": "s3://$S3_LOG_BUCKET/emr/aws-ingestion/",
```

Comment:
Should be `s3://$S3_LOG_BUCKET/emr/analytical-dataset-generation/`

Reply:
done